src/masterlist.py

```diff
@@ -80,6 +80,9 @@
     "Kameroita ei voi olla ikinä liikaa",
     "Passikuvan voi ottaa vaikka puhelimella kotona, siin ei oo kovat vaatimukset",
     "Julkaiskaa enemmän, saa palautetta ja myöhemmin saa myötistellä eiku siis näkee kuinka on kehittyny kuvaajana"
+    "Yksi konsti spiraaliongelmissa on lyödä spiraali napakasti pari kertaa pöytään. Tärähdys saa filmin siellä loppupäässä \"rentoutumaan\" ja taas kulkemaan."
+    "Älä myy vanhaa kuvauskalustoa liian äkkinäisesti pois, se saattaa kaduttaa jälkeenpäin. Filmikaman arvo tuskin laskee vaikka miettisit pari vuotta."
+    "'Buzz, this is Houston. F/2 - 1/160th second for shadow photography on the sequence camera.'\n-Apollo 11 Mission Control, 109:23:26"
```

Comment:
Tähän merkkijonoon tulee nyt heittomerkit lainauksen ympärille. Laita toi mieluummin muodossa ' "lainaus" tekstiä ' niin tulee lainausmerkit myös.

Reply:
Tein näin koska aiempi Petri Mastin lainaus on kirjoitettu tässä muodossa -> konventiot.

Reply:
Ihan ok perustelu